src/test/java/teammates/test/cases/datatransfer/AccountAttributesTest.java

```diff
@@ -115,7 +115,7 @@ public void testGetIdentificationString() {
     @Test
     public void testSanitizeForSaving() {
         AccountAttributes actualAccount = createAccountAttributesToSanitize();
-        AccountAttributes expectedAccount = createAccountAttributesToSanitize();
+        AccountAttributes expectedAccount = actualAccount.getCopy();
```

Comment:
Unrelated change?

Reply:
on one of the ci runs, there was an error where the `expectedAccount` and `actualAccount` differs by the `modifiedDate`.

I traced the creation of the attribute back and since they are supposed to be similar object, this seems to work.

If unrelated, whats the procedue to submit this minor edit?

Reply:
The procedure is to open a new issue for it. However in this case I believe it has already been solved: #8709

Reply:
Ready to merge after you revert this :)